main/nginx/APKBUILD

```diff
@@ -121,6 +121,10 @@ _rtmp_provides="$pkgname-rtmp"  # for backward compatibility
 
 _add_module "http-vod" "1.24" "https://github.com/kaltura/nginx-vod-module"
 
+_add_module "http-geoip2" "3.2" "https://github.com/leev/ngx_http_geoip2_module"
+_http_geoip2_depends="libmaxminddb"
```

Comment:
Why are you depending on libmaxmindb?
abuild will automatically scan elf files for dependencies.

Reply:
I was not aware of that. I noticed the other module packages listing their dependencies, so added the dependency here. If it's not needed, I can remove and squash/rebase. Please confirm.

Reply:
FYR, you can try to build it with abuild -rK (dont forget to cleanup build deps afterwards) and check the contents of

```
pkg/.control.$subpkgname/.PKGINFO
pkg/.control.nginx-mod-http-geoip2/.PKGINFO
```
This should already include the depends to libmaxinddb.
Explicit depends on libs is not needed in normal cases.

Reply:
Yup, you're right. I locally built the package without the dependency listed and `.PKGINFO` lists the libmaxminddb.so and installing the package also installs libmaxminddb.

However, digging through the PKGINFO files, I realized that libmaxminddb is now also "auto detected" as a dependency for the main nginx package. No idea how that happened. libmaxminddb-dev is installed as a build dependency and the module is supposed to be built as a dynamic module, same as the geoip-dev package that is installed as a build dependency for the geoip1 module

Reply:
`checkpkg` says that the main package gets:
``` diff
+usr/lib/nginx/modules/ngx_stream_geoip2_module.so
```

So the automatic package split is not moving all modules to the subpackage. The above file that is left in the main package is introducing the dependency to libmaxminddb.so.

Reply:
@fabled thanks so much for pointing me in the right direction. I'll get that figured out shortly